### PR TITLE
JSON Schema + latest 1.13.1 template

### PR DESCRIPTION
**Pull request by sk-sahu**
Updates the json schema

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] Make sure your code lints (`nf-core lint .`).

### CODE REVIEW

**Comment by apeltzer** [comment]
Would be great to get some reviews in here ....

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,36 +19,38 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       matrix:
-        nxf_ver: ['19.10.0', '']
+        nxf_ver: ['20.04.0', '']
```
Comment (ggabernet) [inline .github/workflows/ci.yml:22]:
```suggestion
        nxf_ver: ['20.10.0', '']
```

Comment (ggabernet) [reply .github/workflows/ci.yml]: I recall there were some issues regarding a variable name with `20.04.0` so I'd recommend going straight to `20.10.0`

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 [![GitHub Actions CI Status](https://github.com/nf-core/scrnaseq/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/scrnaseq/actions)
 [![GitHub Actions Linting Status](https://github.com/nf-core/scrnaseq/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/scrnaseq/actions)
-[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A519.10.0-brightgreen.svg)](https://www.nextflow.io/)
+[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A520.04.0-brightgreen.svg)](https://www.nextflow.io/)
```
Comment (ggabernet) [inline README.md:7]:
```suggestion
[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A520.10.0-brightgreen.svg)](https://www.nextflow.io/)
```

Comment (ggabernet) [reply README.md:7]: Also can run `nf-core bump-version --nextflow .`

File: docs/usage.md [hunk 3]
```diff
@@ -2,55 +2,55 @@
...
+  - [Other command line parameters](#other-command-line-parameters)
+    - [`--outdir`](#--outdir)
+    - [`--email`](#--email)
+    - [`--email_on_fail`](#--email_on_fail)
+    - [`--max_multiqc_email_size`](#--max_multiqc_email_size)
+    - [`-name`](#-name)
+    - [`-resume`](#-resume)
+    - [`-c`](#-c-1)
+    - [`--custom_config_version`](#--custom_config_version)
+    - [`--custom_config_base`](#--custom_config_base)
```
Comment (ggabernet) [inline docs/usage.md:53]:
```suggestion
```

Comment (ggabernet) [reply docs/usage.md:53]: The table of contents should be deleted in `usage.md` as it is created automatically on the website

File: docs/usage.md [hunk 4]
```diff
@@ -2,55 +2,55 @@
 
 ## Table of contents
```
Comment (ggabernet) [inline docs/usage.md:3]:
```suggestion
```

File: nextflow.config [hunk 5]
```diff
@@ -132,7 +170,7 @@ manifest {
   homePage = 'https://github.com/nf-core/scrnaseq'
   description = 'Pipeline for processing of 10xGenomics single cell rnaseq data'
   mainScript = 'main.nf'
-  nextflowVersion = '>=19.10.0'
+  nextflowVersion = '>=20.04.0'
```
Comment (ggabernet) [inline nextflow.config:173]:
```suggestion
  nextflowVersion = '>=20.10.0'
```

**Review by ggabernet — COMMENTED** [review]
Looks good to me, just a couple of minor comments and maybe that the Nextflow version should be bumped to `20.10.0` as otherwise there was a problem with the `$projectDir` variable.

**Comment by apeltzer** [comment]
@sk-sahu some reviews in - you want to have a stab at it?

**Comment by sk-sahu** [comment]
Sure! On it :keyboard:

**Comment by apeltzer** [comment]
Added a tiny bit to the environment.yml to make sure the salmon bit is running through now prperly :-)

**Comment by sk-sahu** [comment]
@apeltzer I think after changing the environment. It fails with `pandoc-citeproc is not available!`. I suppose u fixed it already  

```
Loading required package: alevinQC
  Registered S3 method overwritten by 'GGally':
    method from   
    +.gg   ggplot2
  Loading required package: tximport
  Error in .checkPandoc(ignorePandoc) : pandoc-citeproc is not available!
  Calls: alevinQCReport -> .checkPandoc
  Execution halted
```

**Comment by apeltzer** [comment]
I wanted to - unfortunately this is a bit provlematic now: 

The previous tests were failing because of a known isssue with tbb, which needs to be pinned. 

https://github.com/nf-core/scrnaseq/runs/2169179376

Now after I added this, we have a different issue, because pandoc-citeproc fails. I fear we have to update the entire environment once more now ...?

**Comment by sk-sahu** [comment]
Yeah that's a good idea. Let me update and fix the entire environment

File: environment.yml [hunk 6]
```diff
@@ -6,21 +6,23 @@ channels:
...
-  - gffread=0.11.7
+  - bioconda::bioconductor-alevinqc=1.6.0
+  - bioconda::bioconductor-tximport=1.18.0
+  - bioconda::star=2.7.8a #2.7.3a is broken!
+  - samtools=1.12
+  - gffread=0.12.1
   - bioconda::kallisto=0.46.2
   - bioconda::bustools=0.40.0
   - font-ttf-source-code-pro=2.030 #Required for fonts in alevinQC
+  - pandoc=2.13
```
Comment (ggabernet) [inline environment.yml:27]:
```suggestion
  - conda-forge::pandoc=2.13
```

**Review by ggabernet — COMMENTED** [review]
Let's see if we can fix the tests with this @sk-sahu

**Review by ggabernet — APPROVED** [review]
Looks good! If the tests pass then from my side ready to go 😄

**Review by apeltzer — APPROVED** [review]

**Pull request merged**